demos/assets/ctxMenuFns.js

```diff
@@ -0,0 +1,31 @@
+window.dashCytoscapeFunctions = {};
+
```

Comment:
```suggestion
```

Point is if someone already had something in `window.dashCytoscapeFunctions` we want to keep it - which the form below already handles.